components/scg/src/scg-struct.js

```diff
@@ -415,74 +416,34 @@ function scgScStructTranslator(_editor, _sandbox) {
-
-                    const scAddrEdgeLang = async (linkAddr, keynode) => {
-                        return await window.scHelper.getEdgeLang(linkAddr, keynode);
-                    };
-
-                    const scAddrEdgePng = async (linkAddr, keynode) => {
-                        return await window.scHelper.getEdgePng(linkAddr, keynode);
-                    }
-
-                    const removeLangEdge = async (addrLang) => {
-                        if (!addrLang) return;
-                        await scClient.deleteElements([addrLang]);
-                    };
-
-                    const removeImgEdges = async (addrPng) => {
-                        if (!addrPng) return;
-                        await scClient.deleteElements([addrPng]);
-                    }
-
+            const translateLinks = async function () {
+                const implFunc = async function (link) {
                     const infoConstruction = (link) => {
                         let data = link.content;
                         let type = sc.ScLinkContentType.String;
-                        var keynode = SCWeb.core.Translation.getCurrentLanguage();
-                        let langKeynode = keynode;
-                        let imgKeynode = window.scKeynodes['format_png'];
-
-                        if (link.contentType === 'float') {
-                            keynode = window.scKeynodes['binary_float'];
-                            type = sc.ScLinkContentType.Float;
-                        } else if (link.contentType === 'int8') {
-                            type = sc.ScLinkContentType.Int;
-                            keynode = window.scKeynodes['binary_int8'];
-                        } else if (link.contentType === 'int16') {
-                            type = sc.ScLinkContentType.Int;
-                            keynode = window.scKeynodes['binary_int16'];
-                        } else if (link.contentType === 'int32') {
-                            type = sc.ScLinkContentType.Int;
-                            keynode = window.scKeynodes['binary_int32'];
-                        } else if (link.contentType === 'image') {
-                            let dataStarting = link.content.slice(32);
-                            data = dataStarting.slice(0, dataStarting.length - 14);
-                            type = sc.ScLinkContentType.String;
-                            keynode = window.scKeynodes['format_png'];
+                        let format = window.scKeynodes['format_txt'];
+
+                        if (link.contentType === 'image') {
+                            format = window.scKeynodes['format_png'];
                         } else if (link.contentType === 'html') {
-                            data = link.fileReaderResult;
-                            type = sc.ScLinkContentType.String;
-                            keynode = window.scKeynodes['format_html'];
+                            format = window.scKeynodes['format_html'];
```

Comment:
SCg view for `"format_html"` currently became too slow. It was really fast previously..